packages/core-state/src/defaults.ts

```diff
@@ -3,4 +3,7 @@ export const defaults = {
         maxLastBlocks: 100,
         maxLastTransactionIds: 10000,
     },
+    walletSync: {
+        enabled: false,
```

Comment:
This needs an environment variable to be enabled.

Reply:
oh didnt see that sorry, @rainydio you can fix this in another PR

Reply:
Implemented in #4127